src/Test.FunctionalTests.Sarif/MultitoolCommandLineTests.cs

```diff
@@ -13,6 +13,7 @@ namespace Microsoft.CodeAnalysis.Sarif.Multitool
     public class MultitoolCommandLineTests
     {
         [Fact]
+        [Trait(TestTraits.WindowsOnly, "true")]
```

Comment:
>WindowsOnly [](http://example.com/codeflow?start=26&length=11)

Please file an SDK repo issue to consider authoring Linux equivalents for all tests disabled for that platform.

Reply:
created in the GitHub
[https://github.com/microsoft/sarif-sdk/issues/2449](https://github.com/microsoft/sarif-sdk/issues/2449)